keel-orca/src/main/kotlin/com/netflix/spinnaker/keel/orca/OrcaService.kt

```diff
@@ -89,7 +90,16 @@ data class OrcaException(
   val details: GeneralErrorsDetails
 )
 
+data class KatoException(
```

Comment:
Kato is clouddriver. Better to use `clouddriver` to not spread the confusion of "what the heck is keto?!"

Reply:
You are so right. Will rename it!